### ISSUE TITLE
[Adaptive Card iOS - ProductVideo.json]: Mute or Unmute button on the video player is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#169]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Scenarios' control and activate it.
5. Navigate to 'VideoPlayer.json' under 'Scenarios' list item and activate it.
6. Card will be appear.
7. Perform right swipe and navigate to the video player and double tap to activate it.
8. Now, navigate the control present under player and observe the issue.

**Actual Result:**
While navigating using the swipe gesture, Mute or Unmute button is not accessible through the swipe gesture.

**Expected Result:**
Mute or Unmute button on the video player should be accessible through the swipe gesture.

**User Impact:**
It will impact the screen reader user as they will not able to access the control due to this they will not able to mute or unmute the video.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/57fda79f-9d44-4cbd-9f63-7c02147c12de

### Card JSON

```JSON
NA
```

**Comment by ardlank** [comment]
@GabrielMedAlv @JoshuaChingMSFT please take a look

**Comment by GabrielMedAlv** [comment]
I did research about this issue: The volume component, indeed, doesn't have accessibility. The problem is that the component, is part of the native iOS player AVPlayerView, and is not possible to access the components to modify their individual accessibility. I will continue the investigation, but looks like, by default, won't be able to include accessibility for this component.

**Comment by GabrielMedAlv** [comment]
I validated that the accessibility for this component has been added for iOS 18. Attached evidence. Will proceed to close this issue due to its native component and don't rely on us. https://github.com/user-attachments/assets/399aa6f5-6b73-40bd-973e-ef2213f2da7e

**Comment by vagpt** [comment]
HI @GabrielMedAlv 

Thanks for the updates.

So could you please confirm whether do we need to updated device iOS version to 18.0?
